spec/factories/refund_responses.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  sequence :adyen_refunds_response do
+    OpenStruct.new(
+      response: {
+        "merchantAccount" => "#{SecureRandom.uuid}",
+        "pspReference" => "#{SecureRandom.uuid}",
+        "paymentPspReference" => "#{SecureRandom.uuid}",
+        "status" => "received",
+        "amount" => {
+            "currency" => "CHF",
```

Comment:
Layout/FirstHashElementIndentation: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.